Bubble_Sort/Bubble_Sort.kt

```diff
@@ -0,0 +1,30 @@
+//Function for Bubble Sort
+fun bubbleSort(list: MutableList<Int>):List<Int> {
+    // flag keeps track if any swap happened or not, if swap not happened then array is sorted and it will break out of the loop
+    // and will save time from running for loop unnecessarily
+    var flag: Int = 0
+    for(i in 0 until (list.size - 1)) {
+        for (j in 0 until (list.size - i - 1)) {
+            if(list[j] > list[j + 1]) {
+                var tmp = list[j + 1]
+                list[j + 1] = list[j]
+                list[j]  = tmp
```

Comment:
There are 2 spaces before = in this line.
Please remove one space.

Reply:
Done